astronomer/providers/amazon/aws/example_dags/example_s3.py

```diff
@@ -39,6 +42,17 @@
     "retry_delay": timedelta(seconds=int(os.getenv("DEFAULT_RETRY_DELAY_SECONDS", 60))),
 }
 
+
+def check_dag_status(**kwargs: Any) -> None:
```

Comment:
As this is repeated in multiple DAGs, should we extract it to avoid duplication?

Reply:
I had this thought but have following next thoughts:
1. These are example DAGs and need to be self sufficient
2. We may not want to keep this as a util and release it as part of our providers

Owing to above reasons, I guess the duplication should be okay.